content/changes.md

```diff
@@ -8,6 +8,48 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## January 31, 2024 
+
+### Features 
+ 
+- **Qualified and Learnosity Integration** 
+
+   We've partnered with Learnosity to integrate Qualified coding challenges into Learnosity assessments! Our custom question type enables students to solve project and classic coding challenges in a Learnosity assessment, powered by Qualified's Embed engine. 
+
+  Check our post [Qualified and Learnosity Partnership: Bridging the Skills Gap with Developer Assessments](https://andela.com/blog-posts/qualified-and-learnosity-partnership-bridging-the-skills-gap-with-developer-assessments) to learn more.
+
+### Improvements 
+
+- **Erlang 26** 
+
+    Erlang 26 is now supported within project challenges. 
+
+- **Elixir 1.15** 
+
+    Elixir 1.15 is now supported within project challenges.
+
+- **Add deprecated language version labels** 
+
+    A new "deprecated" label was added to old language versions in the challenge edit and detail views. This helps content managers identify which language versions will be removed in the future. For further information and tips on upgrading your content, please check our [guide for upgrading language versions](https://docs.qualified.io/creating-content/challenges/upgrading-language-versions/#upgrading-language-versions).
+  
+- **Performance issue when loading "Portfolio" tab of report card**
```

Comment:
This should be in a [`### Bug Fixes` section, below the challenges](https://github.com/andela-technology/qualified-docs/blob/a3da79dd9ed37bcd549b9922006d817eab0147e4/scripts/import_product_releases.py#L48).

Reply:
Technically, you could consider any bug fix as an improvement (adding challenges could also be considered an improvement, for that matter), but improvements are things that aren't bugs.

In this case, the performance made that part of the app basically unusable and wasn't intended behavior, so it's a bug.

If you're not sure, you can check the issue type, [which confirms the bug status](https://andela.atlassian.net/browse/QUAL-415).